webpack.conf.js

```diff
@@ -1,28 +1,68 @@
-var webpack = require('webpack');
+var CleanWebpackPlugin = require('clean-webpack-plugin'),
+    config = require('./config.json'),
+    LodashModuleReplacementPlugin = require('lodash-webpack-plugin'),
+    path = require('path'),
+    webpack = require('webpack');
 
 module.exports = {
+    watch: false,
     devtool: 'source-map',
+    context: __dirname,
+    entry: {
+        main: './assets/js/app.js',
+    },
+    output: {
+        filename: 'theme-bundle.[name].js',
+        path: path.resolve(__dirname, "assets/dist")
+    },
     bail: true,
     module: {
-        loaders: [
+        rules: [
             {
                 test: /\.js$/,
-                loader: 'babel',
                 include: /(assets\/js|assets\\js|stencil-utils)/,
-                query: {
-                    compact: false,
-                    cacheDirectory: true,
-                    presets: ['es2015-loose'],
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        compact: false,
+                        cacheDirectory: true,
+                        presets: [
+                            ['latest', {
```

Comment:
@dstaley - yes, there's actually one more PR coming where we'll be using the 'dev' preset instead of 'latest'.  i'm working on it right now.  :)  thanks again for your article!